seed/models/analyses.py

```diff
@@ -75,46 +75,63 @@ def get_property_view_info(self, property_id=None):
         }
 
     def get_highlights(self, property_id=None):
-        from seed.models import PropertyView  # avoiding cyclic dependancy
+        """Get analysis highlights for the overall analysis or for a specific property
+
+        :param property_id: int | None, if provided property-specific highlights
+            from the analysis results are returned. Otherwise highlights from the
+            overall analysis are returned.
+        :return: list[dict{}], a list of highlights as dictionaries, each including
+            a `name` and `value`
+        """
         if self.status < self.COMPLETED:
             return []
+
+        results = {}
         if property_id is not None:
-            analysis_property_view = self.analysispropertyview_set.filter(property=property_id).first()
+            results = self.analysispropertyview_set.filter(property=property_id).first().parsed_results
```

Comment:
https://sentry.io/organizations/nrel/issues/2649369995/?project=1185731

Will this still be an issue after this refactor?  the analyses_property_view set could be None here.

Reply:
Hmm good catch, yeah we should handle that case